opencanary/logger.py

```diff
@@ -153,6 +153,7 @@ def __init__(self, config, handlers, formatters={}):
 
         # Check if ignorelist is populated
```

Comment:
Can we clean this up a little please. And change the below to:
```
       self.ip_ignorelist = config.getVal('ip.ignorelist', default='')
       self.logtype_ignorelist = config.getVal('logtype.ignorelist', default='')
```

Reply:
change done on both (ip and logtype)